spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BinderAwareChannelResolver.java

```diff
@@ -86,8 +93,22 @@ else if (tokens.length != 1) {
 								" [<transport>:]<name>");
 					}
 				}
-				Binder<MessageChannel> binder = binderFactory.getBinder(transport);
-				binder.bindProducer(name, channel, this.producerProperties);
+				String[] outboundChannelNames = null;
+				Properties producerProperties = null;
+				if (this.channelBindingServiceProperties != null) {
+					outboundChannelNames = this.channelBindingServiceProperties.getOutboundChannelNames();
+					// TODO: need the props to return some defaults if not found
```

Comment:
As mentioned on Slack, I strongly believe we should restore global producer and consumer properties in `ChannelBindingServiceProperties`